### PR TITLE
Rz/manage multiply views

### PR DESCRIPTION
**Pull request by pacxiu**
# [Errors/Warnings ManageMultiply](https://app.clubhouse.io/oazo-apps/story/2363/add-extra-errors-warnings-for-manage-multiply-positions)

also covers
# [Third Iteration](https://app.clubhouse.io/oazo-apps/story/2383/other-actions-third-iteration-other-actions-add-errors-warnings)

## Changes 👷‍♀️
- add factory and mocks for `manageMultiplyVault` storybooks
- extract mockExchange data as separate mock
- add animation to `manageMultiply`
- implement OrderInformation similar to `openMultiply` vault
- fix some minor stuff in calculations, eg. calculations not showing on first input data .etc
- add stages storybook
- add separate errors storybook for `Adjust/Other`
- add separate warnings storybook for `Adjust/Other`
  
## How to test 🧪
- inspect UI/Storybook for manage multiply
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

File: features/manageMultiplyVault/components/ManageMultiplyVaultDetails.tsx [hunk 1]
```diff
@@ -94,29 +93,35 @@ function ManageMultiplyVaultDetailsSummary({
...
   } = props
-  const { t } = useTranslation()
   const openModal = useModal()
-  const collRatioColor = getCollRatioColor(props, collateralizationRatio)
-  const collRatioNextPriceColor = getCollRatioColor(props, collateralizationRatioAtNextPrice)
   const afterCollRatioColor = getCollRatioColor(props, afterCollateralizationRatio)
   const afterPillColors = getAfterPillColors(afterCollRatioColor)
   const showAfterPill = !inputAmountsEmpty && stage !== 'manageSuccess'
 
+  // TO DO CALCS
```
Comment (johnnieskywalker) [inline features/manageMultiplyVault/components/ManageMultiplyVaultDetails.tsx:108]: We should have one way of putting To do in code so it's easy to find later by one format either `// TO DO` or `// TODO` etc.But so far we have not agreed on that so it's just an opinion

File: features/manageMultiplyVault/stories/ManageMultiplyVaultWarningsAdjustPosition.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+  },
+  priceInfo: {
+    collateralChangePercentage: new BigNumber('-0.4'),
+  },
+  proxyAddress,
+})({
+  requiredCollRatio: new BigNumber(2.8),
+})
+
+// export const GeneratingAllDaiFromIlkDebtAvailable = manageMultiplyVaultStory({
```
Comment (johnnieskywalker) [inline features/manageMultiplyVault/stories/ManageMultiplyVaultWarningsAdjustPosition.stories.tsx:155]: I guess this is going to be uncommented once implementation will be fully done but as it's work in progress PR to feature branch it's ok too

**Review by johnnieskywalker — DISMISSED** [review]
Added minor comments but as this is work in progress, and pull request is to feature branch IMHO we can merge it

File: features/manageMultiplyVault/manageMultiplyVaultCalculations.ts [hunk 3]
```diff
@@ -488,7 +487,7 @@ export function applyManageVaultCalculations(
     maxGenerateAmount,
   }
 
-  if (!marketPrice || !marketPriceMaxSlippage || inputAmountsEmpty) {
+  if (!marketPrice || !marketPriceMaxSlippage) {
```
Comment (paszkowskiDamian) [inline features/manageMultiplyVault/manageMultiplyVaultCalculations.ts:490]: We cannot start calculations when the slider is not set, one solution would be to set some default `requiredCollateralizationRato` 

This is the reason why we get `NaN`s

Comment (pacxiu) [reply features/manageMultiplyVault/manageMultiplyVaultCalculations.ts:490]: When it was present we had such flickering in UI for calculations:
https://user-images.githubusercontent.com/19193499/129314569-3dd06b01-08a1-4c5a-a354-bdbfa3b8144f.mp4

**Pull request merged**